### PR TITLE
Updating the documentation of MAPE

### LINKED ISSUE #15599: Error in the documentation of tf.keras.losses.MeanAbsolutePercentageError
# Description of the issue

The documentation of MAPE wrongly states that it is defined as `loss = 100 * abs(y_true - y_pred) / y_true` (see https://github.com/keras-team/keras/blob/2c48a3b38b6b6139be2da501982fd2f61d7d48fe/keras/losses.py#L372-L429).

However, it is implemented the right way (see this function, both documentation and computation are good : https://github.com/keras-team/keras/blob/2c48a3b38b6b6139be2da501982fd2f61d7d48fe/keras/losses.py#L1349)

# Proposed fix

Change `loss = 100 * abs(y_true - y_pred) / y_true` to `loss = 100 * abs((y_true - y_pred) / y_true)` to match the true formula used in computation. I can make the PR if you think it's relevant!

# Addendum

I also noticed that the implementation uses an epsilon in the denominator to avoid 0-division, which makes sense (same as in scikit-learn, for instance, see https://github.com/scikit-learn/scikit-learn/blob/0d378913b/sklearn/metrics/_regression.py#L286). I think it would be worthwhile to mention it also in keras' doc.

### PR DESCRIPTION
**Pull request by sanatmpa1**
Correcting the formula in documentation of MAPE and also adding a note about using epsilon in denominator.

Fixes #15599

### CODE REVIEW

File: keras/losses.py [hunk 1]
```diff
@@ -372,7 +372,9 @@ def __init__(self,
 class MeanAbsolutePercentageError(LossFunctionWrapper):
   """Computes the mean absolute percentage error between `y_true` and `y_pred`.
 
-  `loss = 100 * abs(y_true - y_pred) / y_true`
+  `loss = 100 * abs((y_true - y_pred) / y_true)`
+  
+  Note that to avoid division by zero error, a small value epsilon is added to the denominator.
```
Comment (fchollet) [inline keras/losses.py:377]: "Note that to avoid dividing by zero, a small epsilon value is added to the denominator."

Comment (sanatmpa1) [reply keras/losses.py]: @fchollet, I have modified as per your advise, Please take a look on it.

**Review by fchollet — APPROVED** [review]
LGTM
